common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -73,6 +73,7 @@ class ProjectCard extends React.PureComponent<Props, State> {
           {this._renderTitleAndIssue()}
           {this._renderProjectDescription()}
           {this._renderSkillsNeeded()}
+          {this._renderCardOperation()}
```

Comment:
Making a note for review: I need to test the project cards on homepage/about projects page due to this change, it's possible the conditional rendering needs to be here due to moving renderCardOperation up here.